results/intfloat__multilingual-e5-small/ARCChallenge.json

```diff
@@ -0,0 +1,158 @@
+{
+  "dataset_revision": "c481e0da3dcbbad8bce7721dea9085b74320a0a3",
+  "evaluation_time": 2.7299964427948,
+  "kg_co2_emissions": null,
+  "mteb_version": "1.12.27",
+  "scores": {
+    "test": [
```

Comment:
Based on the directory structure (lacking model revision), I believe that this is run on an older MTEB version.

Reply:
Ahh right - good catch, we might need to add a test for that

Reply:
yes, can't remember the exact case but for multiligual-e5-small specifically, the specified model version was not able to load when I was putting together the PR. So I had to modify the code a bit to load the latest version by default in a different branch and copy to results in. All other models I tested were running fine though. @KennethEnevoldsen @isaac-chung

Reply:
Let us just remove the results from the main branch and rerun it using the latest version to ensure no oddities.

Reply:
Opened https://github.com/embeddings-benchmark/mteb/issues/955 and will clean up results in that PR too.